test/browser_reporting.js

```diff
@@ -29,7 +32,15 @@ function maybeReportResultToServer(result) {
   }
 }
 
-function reportErrorToServer(message) {
+function reportStderrToServer(message) {
+  if (typeof ENVIRONMENT_IS_NODE !== 'undefined' && ENVIRONMENT_IS_NODE) {
+    err(message);
+  } else {
+    fetch(`${reportingURL}?stderr=${encodeURIComponent(message)}`);
```

Comment:
Is there any limit on how much data you can send to the server in a URL query parameter  this way? Should we have some kind of check, or something more robust like POST?

Reply:
Great idea yes.

Last time I remember trying to enable this by default i ran into issues that could be explained by that perhaps.

Reply:
If its OK with you can leave that as a followup?

Reply:
Yeah, as long as you don't think this would cause any new failures. I guess not since this is still off by default?

Reply:
Indeed, its off by default for now.

Reply:
I have a followup prepared that does this.